app/shared/video-player/desktop/ui/VideoPlayer.desktop.kt

```diff
@@ -147,19 +141,10 @@ class VlcjVideoPlayerState(parentCoroutineContext: CoroutineContext) : PlayerSta
         }
     }
 
-    override fun getScreenshotPath(): Path? {
-        var screenshotPath: Path? = null
-        if (Platform.currentPlatform.isDesktop()) {
-            screenshotPath = Path.of(System.getProperty("user.home")).resolve("Pictures").resolve("Ani")
-            screenshotPath.createDirectories()
-        }
-        return screenshotPath
-    }
-
     override fun saveScreenshotFile(filename: String) {
-        val screenshotPath = getScreenshotPath()
-        // return when screenshot path is null
-        if (Objects.isNull(screenshotPath)) return
+        val screenshotPath: Path? =
+            Path.of(System.getProperty("user.home")).resolve("Pictures").resolve("Ani")
+        screenshotPath?.createDirectories()
         val filePath = screenshotPath?.resolve(filename)
         player.snapshots().save(filePath?.toFile())
```

Comment:
```suggestion
        val screenshotPath = Path.of(System.getProperty("user.home")).resolve("Pictures").resolve("Ani")
        screenshotPath.createDirectories()
        val filePath = screenshotPath.resolve(filename)
        player.snapshots().save(filePath.toFile())
```

`resolve` 不会返回 `null` 吧

Reply:
不会